data/username_checker.json

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
Compress the JSON file.
Where is the source of this file? take place the source of JSON file in the meta variable.

Reply:
> Where is the source of this file? take place the source of JSON file in the meta variable.
I did not understand this.

Reply:
Also will compress the JSON file after I have added all the sites.

Reply:
Did you write the JSON file by yourself?

Reply:
here is a list of social networks I am referring to https://github.com/sherlock-project/sherlock/blob/master/sites.md

Reply:
Therefore, write this line in the meta:
`'sources': ('https://github.com/sherlock-project/sherlock',),`